### ISSUE TITLE
conv1D missing as a global function

### ISSUE THREAD

**Issue by SamPruden** [tensorflow/swift-apis#528]
`Conv1D` is available as a layer which is implemented via `conv2D`, but `conv1D` is not available as a global function. I'm using convolution for data processing outside of the net, so it would be useful to have. Is there interest in getting this added?

I've just nabbed and tweaked the implementation from `Conv2D` for now.

```Swift
@differentiable(wrt: (input, filter))
func conv1D<Scalar: TensorFlowFloatingPoint>(
  _ input: Tensor<Scalar>,
  filter: Tensor<Scalar>,
  stride: Int = 1,
  padding: Padding = .valid,
  dilation: Int = 1
) -> Tensor<Scalar> {
  return conv2D(
    input.expandingShape(at: 1),
    filter: filter.expandingShape(at: 0),
    strides: (1, 1, stride, 1),
    padding: padding,
    dilations: (1, 1, dilation, 1)
  ).squeezingShape(at: 1)
}
```

I'm pretty new to playing with Swift and TensorFlow and I'm definitely not a linear algebra expert so I don't want to take responsibility for claiming this is a good implementation, but it looks like it could just be dropped into `Operators/NN.swift` with appropriate comments and that would be it done. I can't see any test coverage for the other similar global functions.

If we want this done and that's all there is to it I could make a pull, but I'd be happier for somebody else to just drop it in. I don't even have a proper Swift setup here at the moment (I'm just using Colab) and I feel a little uncomfortable putting a pull together without even confirming it builds, so if somebody else wants to do it that would be great.

**Comment by ocampor** [comment]
I will take a look to this one.

**Comment by saeta** [comment]
I believe this is now fixed due to #549. Thanks! -Brennan
